public/bz_support.js

```diff
@@ -83,3 +83,7 @@ function bzRetainedInfoSetup() {
   }
 }
 
+if(window != window.top) {
```

Comment:
On Tue, May 17, 2016 at 11:23:45AM -0700, Brian Sadler wrote:
> > +if(window != window.top) {
> 
> Was this change intentional?

Yes, though not necessarily related to the rest - this little
javascript gives me the CSS hooks needed for a separate PR in the
-js-css repo for the iframing stuff.

Just adding the class is the only change needed on this level for
the basic iframe feature.